SocketRocket/SRWebSocket.h

```diff
@@ -303,6 +317,18 @@ extern NSString *const SRHTTPResponseErrorKey;
 - (BOOL)sendDataNoCopy:(nullable NSData *)data error:(NSError **)error NS_SWIFT_NAME(send(dataNoCopy:));
 
 /**
+ Send binary data to the server, without making a defensive copy of it first.
+ 
+ @param data  Data to send.
+ @param complete The call back of send result.
+ If an error occurs, this block will invoked with an `NSerror` object cantaining information about the error; Otherwise
+ this block will invoked with `nil`.
+ 
+ @return `YES` if the string was scheduled to send, otherwise - `NO`.
+ */
+- (BOOL)sendDataNoCopy:(nullable NSData *)data completed:(SRSendCompleteBlock)complete;
```

Comment:
The above code block also ensures that everyone can pass nil to the completion argument, since it looks like you are validating it anyway.